TC Coding Challenge 3 Submission

halbright19 opened the pull request.

halbright19 committed:
Update ATM.java

halbright19 committed:
withdraw and deposit fully tested

Commit:
Full code coverage in ATM.java

halbright19 committed:
Delete .project

halbright19 committed:
Restore deleted file

pauquette commented:
Make sure your tests also cover CurrencyExchanger.java and UnsupportedCurrencyCodeException.java as well for 100% code coverage

halbright19 committed:
Full code coverage

Full code coverage in ATM.java, CurrencyExchanger.java, and UnsupportedCurrencyCodeException.java. The only code that is not covered is in the try-catch statements in Test.java

halbright19 commented:
Now the only code not covered is try-catch statements in Test.java

Commit:
updated exception tests
